spec/features/post_index.spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :feature do
```

Comment:
There are some tests failing. Kindly see the screenshot below and fix them all.

![specs](https://user-images.githubusercontent.com/35504149/200443539-9da1aa34-72c3-485d-85f8-84e3fc6b07e0.png)

![specs2](https://user-images.githubusercontent.com/35504149/200443556-f1c4f3fd-8bde-4b16-8098-faad64a6574e.png)